src/cargo/util/profile.rs

```diff
@@ -22,6 +22,7 @@ pub struct Profiler {
 }
 
 fn enabled_level() -> Option<usize> {
+    #[allow(clippy::disallowed_methods)]
     env::var("CARGO_PROFILE").ok().and_then(|s| s.parse().ok())
 }
```

Comment:
What about the alternative solution I mentioned above?

> Alternatively, we could have a module `env_var`, putting all hardcoded env var there and giving each of them a dedicated function. So that we can document them in one place.

Reply:
Guess I hadn't read far enough :)

I think splitting these out into functions makes sense.  Whether to centralize them I can go either way on since that also means we are putting a bunch of unrelated items together far away from their use.

Reply:
Yeah. They are not required to be centralized. Putting them in one place is just an attempt to have less `#[allow(…)]` scattered. The callsite would look like `crate::allowed_env::cargo_profile()`.

I am fine with both, if we decide to adopt `disallowed_methods` to help us.